### PR TITLE
file name change to test_sse_s3_kms_with_vault.py

### PR DESCRIPTION
**Pull request by viduship**
file name change to test_sse_s3_kms_with_vault.py
Signed-off-by: viduship <vimishra@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by udaysk23 — APPROVED** [review]
Looks good to merge once below PR get merged into ceph-qe-script
https://github.com/red-hat-storage/ceph-qe-scripts/pull/372

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**
